includes/class-wc-admin-reports-segmenting.php

```diff
@@ -342,17 +342,31 @@ protected function set_all_segments() {
 			$segment_objects = wc_get_products( $args );
 
 			foreach ( $segment_objects as $segment ) {
-				$id = $segment->get_id();
-				$segments[] = $id;
+				$id                    = $segment->get_id();
+				$segments[]            = $id;
 				$segment_labels[ $id ] = $segment->get_name();
 			}
 		} elseif ( 'category' === $this->query_args['segmentby'] ) {
-			$categories = get_categories(
```

Comment:
Instead of the query, does the `include` query argument work instead? It looks like get_categories should support it.

https://github.com/woocommerce/wc-admin/blob/94b296b3460d11d3c14a7eac6349693072315c6b/includes/class-wc-admin-reports-segmenting.php#L314-L318

Reply:
Or is this the problem you mentioned below?